docker/android/android/__init__.py

```diff
@@ -0,0 +1,15 @@
+import sys
+
+# we run this script once every build, and we'd rather
+# have much smaller image sizes, so copying without
+# any bytecode is a better idea.
+sys.dont_write_bytecode = True
```

Comment:
There's no reason not to have this, since we're almost entirely running this script **inside** the Docker image and only running it once, so any nearly insignificant performance improvements during our git hooks (which won't test Python unless you explicitly opt-in) won't matter.